### ISSUE TITLE
Assign button and Invite Colleague discussion in 0.9.0 not working in IE 11; console reports "show_invite_modal" in undefined.

### ISSUE THREAD

**Issue by gregelin** [GovReady/govready-q#664]
Clicking on "Assign" button in a question does not pop-up invite modal in 0.9.0.dev 
Invite colleague to discussion link is also failing to popup modal in IE 11.

Issue encountered in Internet Explorer 11 version 11.829.17134.0 / 11.0.130

Important to fix this issue because many gov employees will use IE 11.

**Comment by gregelin** [comment]
In template `project.html` the Invite button xx calls Javascript function `invite_user_into_project` which executes. Within `invite_user_into_project` the function `show_invite_modal` is called and it appears that is throwing the error.

The function `show_invite_modal` is defined in `siteapp/static/js/main.js` and `main.js` is getting downloaded.

**Comment by gregelin** [comment]
Could this code block within `show_invite_modal` function be causing the function to not process correctly? There is no '{' after ` if (s.val() == data.user_id)` followed by an indented line.

```
  if (data.user_id) {
    s.val(data.user_id);
    if (s.val() == data.user_id) // valid?
      s.prop('disabled', true);
  }
```

**Comment by gregelin** [comment]
Chrome Javascript console displays `window.textarea_ctrlenter_handler` and `window.show_invite_modal` indicating `js/main.js` is being correctly loaded and parsed.

IE11 Javascript console does **not** display `window.textarea_ctrlenter_handler` and `window.show_invite_modal` indicating `js/main.js` is **not** being correctly loaded and parsed.

**Comment by gregelin** [comment]
IE11 is reporting SCRIPT1002: Syntax error for `main.js` in line 222, 18.

```
      var tx = t**1.5;
```

The error may be that IE11 does not like `**` for power function and wants to use `Math.pow($1,$2)`. Fortunately Chrome and Firefox accepts `Math.pow($1,$2)`, so let's change the power functions to `Math.pow`.

**Comment by gregelin** [comment]
This issue is fixed. The problem was  IE11 does not like ** for power function and wants to use Math.pow($1,$2). 

Fixed in in 0.9.0.dev commit 0cdca7cbd4f9fe2e408b7b2e63b93df8c5d447a6 and 42089361fbaa099ee0366fad37d406e72d71b79e
Fixed in 0.9.0-mesosphere commit 069e48a39a8e153f4cf8ec1012b526f51c208781
